lighthouse-cli/test/smokehouse/byte-efficiency/expectations.js

```diff
@@ -13,17 +13,16 @@ module.exports = [
     initialUrl: 'http://localhost:10200/byte-efficiency/tester.html',
     url: 'http://localhost:10200/byte-efficiency/tester.html',
     audits: {
-      // TODO: re-enable once CSS protocol has stabilized
-      // 'unused-css-rules': {
-      //   score: false,
-      //   extendedInfo: {
-      //     value: {
-      //       results: {
-      //         length: 2
-      //       }
-      //     }
-      //   }
-      // },
+      'unused-css-rules': {
+        score: '<100',
```

Comment:
sure, the score gets bucketed by the generic byte efficiency audit though so it's not that useful. I added an assertion on the "wastedKb" instead which is a little more explicit and useful